### ISSUE TITLE
Alpine agent - Unable to send file 'merged.mg' to agent ID

### ISSUE THREAD

**Issue by Loic-Hakisa** [wazuh/wazuh#15814]
Hello,

It's my first time trying to use Wazuh on AWS ECS infra with alpine container, already use OSSEC on a beanstalk infra.

I used the recent alpine package (v4.5.0) with Wazuh manager App version: 4.3.10, installed using : 
https://packages.wazuh.com/4.3/wazuh-install.sh

I receive alerts for the manager but none from the agents.

The alpine agent is in status connected in the manager Dashboard, but don't work, I found some error logs : 

> 2022/12/29 13:12:30 wazuh-authd: INFO: New connection from 10.0.4.60
> 2022/12/29 13:12:30 wazuh-authd: INFO: Received request for a new agent (ip-10-0-4-60.eu-central-1.compute.internal) from: 10.0.4.60
> 2022/12/29 13:12:30 wazuh-authd: INFO: Agent key generated for 'ip-10-0-4-60.eu-central-1.compute.internal' (requested by any)
> 2022/12/29 13:12:40 wazuh-remoted: INFO: (1409): Authentication file changed. Updating.
> 2022/12/29 13:12:40 wazuh-remoted: INFO: (1410): Reading authentication keys file.
> 2022/12/29 13:13:11 wazuh-remoted: WARNING: Package dropped. Could not append data into buffer.
> 2022/12/29 13:13:11 wazuh-remoted: WARNING: (1246): Unable to send file 'merged.mg' to agent ID '009'.
> 2022/12/29 13:13:21 wazuh-remoted: WARNING: Package dropped. Could not append data into buffer.
> 2022/12/29 13:13:21 wazuh-remoted: WARNING: (1246): Unable to send file 'merged.mg' to agent ID '009'.
> 2022/12/29 13:13:31 wazuh-remoted: WARNING: Package dropped. Could not append data into buffer.
> 2022/12/29 13:13:31 wazuh-remoted: WARNING: (1246): Unable to send file 'merged.mg' to agent ID '009'.
> 2022/12/29 13:13:41 wazuh-remoted: WARNING: Package dropped. Could not append data into buffer.
> 2022/12/29 13:13:42 wazuh-remoted: WARNING: Package dropped. Could not append data into buffer.
> 2022/12/29 13:13:42 wazuh-remoted: WARNING: (1246): Unable to send file 'merged.mg' to agent ID '009'.

![image](https://user-images.githubusercontent.com/63661291/209979513-57711204-9ce5-4a84-9a41-a6bd24c1888d.png)

Do you have any idea of the problem ? What can i check is it the version mismatch between agent and manager ?

Best regards

**Comment by c-bordon** [comment]
Hello, the error is possibly due to the fact that you are using a version that is currently under development, which is not recommended. I recommend you to build a package from our last released version, v4.3.10, I can share a Dockerfile with all the necessary steps for you to build the 4.3.10 package:
```console
FROM alpine:latest
RUN apk update && apk add make gcc automake autoconf libtool curl openssl openssl-dev libgcc cmake gnupg perl fakeroot brotli automake autoconf libtool gawk libsigsegv nodejs build-base python3 libc-dev gettext-dev zip procps --update alpine-sdk
RUN apk --no-cache add ca-certificates wget
RUN wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://alpine-pkgs.sgerrand.com/sgerrand.rsa.pub
RUN wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.35-r0/glibc-2.35-r0.apk
RUN apk add --force-overwrite glibc-2.35-r0.apk
RUN apk add linux-headers && ln -s /usr/include/linux/a.out.h /usr/include/a.out.h
RUN wget https://github.com/wazuh/wazuh/archive/refs/heads/AlpineCompileForCommunity-4.3.10.zip
RUN unzip AlpineCompileForCommunity-4.3.10.zip
RUN cd wazuh-AlpineCompileForCommunity-4.3.10/src && make deps TARGET=agent EXTERNAL_SRC_ONLY=1
RUN addgroup -S wazuh && adduser -S wazuh
RUN cd wazuh-AlpineCompileForCommunity-4.3.10/src && make TARGET=agent
RUN cd wazuh-AlpineCompileForCommunity-4.3.10/ ./install.sh
```
With version 4.4.0 of Wazuh we will provide an installable package for Alpine.

**Comment by Loic-Hakisa** [comment]
Thanks @c-bordon , I confirm using these steps it's working,
But the building time is very long, need to find how to use with our workflow.

Any idea when 4.4.0 will be released ? I saw Q1 2023

Happy holidays and new year !
